opds_import.py

```diff
@@ -825,6 +825,10 @@ def extract_link(cls, link_tag, feed_url=None):
         rel = attr.get('rel')
         media_type = attr.get('type')
         href = attr.get('href')
+        if not href or not rel:
+            # The link exists but has no destination, or no specified
+            # relationship to the entry.
+            return None
```

Comment:
This stops `<link>` tags with empty `href` or `rel` fields from showing up as LinkData objects.